File: java/src/list_number/MaxNumber.java
package list_number;

public class MaxNumber {
    public static void main(String[] args) {
        int[] numbers = {1, 2, 3, 4, 9, 5, 6};

        if (numbers.length == 0) {
            System.err.println("Array is empty");
            return;
        }

        int max = numbers[0];

        for (int num : numbers) {
            if (num > max) {
                max = num;
            }
        }

        System.out.printf("The maximum number is: %d\n", max);
    }
}

File: go/list_number/MaxNumber.go
package list_number

import (
"fmt"
"log"
)

func main() {
	numbers := []int{1, 2, 3, 4, 9, 5, 6}

	if len(numbers) == 0 {
		log.Fatal("Slice is empty")
	}

	max := numbers[0]

	for _, num := range numbers {
		if num > max {
			max = num
		}
	}

	fmt.Printf("The maximum number is: %d\n", max)
}
